docs/.templates/integration/schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories_list": {
+            "type": "object",
+            "description": "A list defining all the available categories for the integrations",
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of the category"
+                            },
+                            "priority": {
+                                "type": "string",
+                                "description": "Priority of the category"
+                            },
+                            "id": {
+                                "type": "string",
+                                "description": "ID of the category that will be used by integrations"
+                            },
+                            "parent": {
+                                "type": [
+                                    "null",
+                                    "string"
+                                ],
+                                "description": "The category's parent ID, make it null if no parent"
```

Comment:
- parent => parent_id
- more info to `description`. It is not clear why this property is needed/how it will work.

Reply:
John explained it [here](https://github.com/netdata/netdata/pull/15302/files#r1250981042), but I will do my best to try and explain it on the schema

Reply:
I see, thanks. I think a good description is needed here.

Q: Is having a flat list required? Why don't we have a sub-category inside a category?

Reply:
I think it is not a problem given no-one will edit this JSON, nor humans will read it. Ideally categories will be handled in a structured way in some CSV somewhere.

Reply:
Ok, but I think it is better to have categories nested because you can easily see the hierarchy and spot any errors (made by your script) if any. As it is suggested (flat list with parent ids) it is not possible and you need to actually build your menu (website) to see if the hierarchy in integrations is wrong.

Reply:
But the structured place I mention will be the place to check the categories, ordering and such, this will be just a code representation, maybe @hugovalente-pm and @sashwathn can provide some thoughts too

Reply:
>  this will be just a code representation

I think it is not only representation but will be used by something to build things. Otherwise, why would we need hierarchical categories in this file?

Reply:
@kapantzak will you then need to have these nested? I went with this flat structure from your idea, but also because this is not going to be edited by a person.

I understand the nesting mechanic, but if we have a spot to manage them, it doesn't matter how they appear in here, I don't expect someone to come into the JSON to understand the structure, he will go to the place where every integration will be grouped, in a human readable way, that will be the source for this fields